src/Sulu/Bundle/TagBundle/Resources/config/lists/tags.xml

```diff
@@ -85,6 +93,11 @@
 
                 <joins ref="creator"/>
             </field>
+
+            <filter type="selection">
```

Comment:
same as above

Reply:
Because it seems that this is the only file where this property exists :slightly_smiling_face: